README.md

```diff
@@ -27,11 +27,144 @@ Notes:
 `vp_hash`
 OPTIONAL. Hash value of `vp_token` that represents the W3C VP. Its value is the base64url encoding of the left-most half of the hash of the octets of the ASCII representation of the vp_token value, where the hash algorithm used is the hash algorithm used in the alg Header Parameter of the ID Token's JOSE Header. For instance, if the alg is RS256, hash the vp_token value with SHA-256, then take the left-most 128 bits and base64url encode them. The vp_hash value is a case sensitive string.
 
-# Examples
+# Frontchannel
+This section illustrates the protocol flow for the case of communication through the front channel only (like in SIOP).
+## Authentication request
+
+The following is a non-normative example of how an RP would use the `claims` parameter to request claims in the `vp_token`:
+
+```
+  HTTP/1.1 302 Found
+  Location: openid://?
+    response_type=id_token
+    &client_id=https%3A%2F%2Fclient.example.org%2Fcb
+    &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+    &scope=openid
+    &claims=...
+    &state=af0ifjsldkj
+    &nonce=n-0S6_WzA2Mj
+    &registration_uri=https%3A%2F%2F
+      client.example.org%2Frf.txt%22%7D
+      
+```
+### claims parameter (simple)
+
+In its simplest form, the RP just asks the OP to provide a VP or VC containing a set of claims in the response by listing those claims underneath `vp_token`.  
+
+```
+{
+    "id_token": {
+        "acr": null
+    },
+    "vp_token": {
+        "given_name": null,
+        "family_name": null,
+        "birthdate": null
+    }
+}
+```
+
+### claims parameter (with identity assurance requirements)
+
+The TP may also request claims fulfilling the requirtements of a certain trust framework and identity assurance level. It does so by adding a `verified_claims` claims element to the `vp_token` section, which defines the verification requirements and lists the respective End-User claims. 
+Note: this syntax was adopted from the OpenID Connect for Identity Assurance spec (https://openid.net/specs/openid-connect-4-identity-assurance-1_0.html).
+
+```
+{
+    "id_token": {
+        "acr": null
+    },
+    "vp_token": {
+        "verified_claims": {
+            "verification": {
+                "trust_framework": {
```

Comment:
@tlodderstedt In that case the `eidas` trust framework would define which `issuer`DIDs are trustworthy. Is this correct?

Reply:
yes. There is a need for an authoritative list of such issuers similar to https://ec.europa.eu/cefdigital/wiki/display/EIDCOMMUNITY/Overview+of+pre-notified+and+notified+eID+schemes+under+eIDAS and https://webgate.ec.europa.eu/tl-browser/. ESSIF seems to be heading there.